test/built-ins/Function/prototype/toString/async-arrow-function.js

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-async-arrow-function-definitions-runtime-semantics-evaluation
+description: Function.prototype.toString on an async arrow function
+---*/
```

Comment:
I believe this is the only part here in this PR missing a `features: [async-functions]`.